app/book.py

```diff
@@ -0,0 +1,7 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Book:
+    title: str
+    content: str
```

Comment:
Why not using basic python classes?)

Reply:
I thought it would be easier, especially since they fit well here